.github/workflows/container_app_push.yml

```diff
@@ -0,0 +1,102 @@
+---
+name: Application Container Image
+
+on:
+    # We are deliberately *not* running on push events here to avoid double runs.
+    # Instead, push events will trigger from the base image and maven unit tests via workflow_call.
+    workflow_call:
+    pull_request:
+        branches:
+            - develop
+            - main
```

Comment:
This should be master instead of main, right?

Reply:
Good catch, done.